src/citrine/informatics/predictors/auto_ml_predictor.py

```diff
@@ -54,10 +88,12 @@ def __init__(self,
                  output: Descriptor = None,
                  outputs: List[Descriptor] = None,
                  inputs: List[Descriptor],
+                 estimators: Optional[Set[AutoMLEstimator]] = None,
                  training_data: Optional[List[DataSource]] = None):
         self.name: str = name
         self.description: str = description
         self.inputs: List[Descriptor] = inputs
+        self.estimators: Set[AutoMLEstimator] = estimators or {AutoMLEstimator.RANDOM_FOREST}
```

Comment:
This replaces an explicitly passed empty set with the default value.  Intended?

Reply:
Hmm, passing an empty set is guaranteed to fail in the backend, just later on.

I suppose we could allow it here, but it will always result in an invalid module.

Reply:
Ehh I think it makes sense to coerce to a valid default here.